NEWS

```diff
@@ -1,3 +1,20 @@
+09-Aug-2018 IGNITION v0.27.0
+
+    Features:
+
+    - Ignition is now built as a Position Independent Executable (PIE)
```

Comment:
Bullets are not properly indented.

Reply:
The bullets were correctly indented, but the headings were not. Fixed!